offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/offergeneration/QuotationDetails.groovy

```diff
@@ -123,8 +123,8 @@ class QuotationDetails {
         addSubTotalPrices(productGroup, subTotal)
     }
 
-    private static String createDiscountDescription() {
-        return "Quantity discount."
+    private String createDiscountDescription(double quantity, String unit, double discountPercentage) {
```

Comment:
This can be static, I think but `private static`

Reply:
no, because it uses the item number which is a classfield